docs/scripts/math.js

```diff
@@ -0,0 +1,33 @@
+class MathController extends Stimulus.Controller {
+  static get targets() {
+    return ['answer', 'equation', 'result']
+  }
+
+  connect() {
+    this.a = this.randomNumber()
+    this.b = this.randomNumber()
+    this.equationTarget.textContent = `${this.a} * ${this.b}`
+  }
+
+  randomNumber() {
+    return Math.floor(Math.random() * 10) + 1
+  }
+
+  greet() {
```

Comment:
How about we change the name of the method?

```suggestion
  submit() {
```

Reply:
Changed this plus another reference